### PR TITLE
Refactor and Standardize the Examplescript

### PR DESCRIPTION
**Pull request by amayer5125**

### CODE REVIEW

File: template/examplescript/examplescript.sh [hunk 1]
```diff
@@ -39,58 +39,55 @@ cat <<END >/etc/apache2/sites-available/$WEBSITE.conf
...
-"
+mysql -uroot -p$DB_PASSWORD -e "CREATE DATABASE wordpress;"
+mysql -uroot -p$DB_PASSWORD -e "CREATE USER '$DBUSER' IDENTIFIED BY '$DBUSER_PASSWORD';"
 mysql -uroot -p$DB_PASSWORD -e "GRANT ALL PRIVILEGES ON wordpress.* TO '$DBUSER';"
 
-service mysql restart
- 
-#installing php
-apt-get install php7.0 php-pear libapache2-mod-php7.0 php7.0-mysql -y php-gd
+systemctl mysql restart
```
Comment (Qtrain) [inline template/examplescript/examplescript.sh:87]: should be systemctl restart mysqld

Comment (amayer5125) [reply template/examplescript/examplescript.sh]: Good catch. Updated.

**Comment by amayer5125** [comment]
Resolving conflicts.

**Comment by amayer5125** [comment]
Requested changes and rebase are complete.

**Pull request merged**